ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/config-comment.j2

```diff
@@ -0,0 +1,16 @@
+{#
+ Copyright (c) 2023-2024 Arista Networks, Inc.
+ Use of this source code is governed by the Apache License 2.0
+ that can be found in the LICENSE file.
+#}
+{# eos - config comment #}
+{% if config_comment is arista.avd.defined %}
+!
+{%     set multiline_comment = config_comment.split('\n') %}
+{%     for comment in multiline_comment %}
+{# To avoid empty sting #}
+{%         if comment %}
+!{{ comment }}
+{%         endif %}
```

Comment:
I don't think we need to check for empty lines. If someone puts in an empty line, we should print it.
```suggestion
!{{ comment }}
```

Reply:
Fixed!